metapool/tests/test_plate.py

```diff
@@ -261,6 +269,15 @@ def test_validate_plate_metadata_full(self):
         pd.testing.assert_frame_equal(validate_plate_metadata(self.metadata),
                                       expected)
 
+    def test_validate_plate_metadata_full_extra_column(self):
+        for item in self.metadata:
+            item['NewColumn'] = 'NewValue'
```

Comment:
You could remove these lines and just add it to the dataframe after line 276, like:
expected['NewColumn'] = 'NewValue'

Reply:
Actually, the value needs to be added to the dictionaries in self.metadata as well. The objective of this test is to show that an extra unexpected column is not scrubbed from the results of validate_plate_metadata().